content/applications/marketing/marketing_automation/getting_started/first_campaign.rst

```diff
@@ -1,70 +1,113 @@
 
-On the blank marketing automation campaign page, the following smart buttons and fields are
-available:
-
-.. image:: first_campaign/marketing-template-sample.png
+.. image:: first_campaign/blank-marketing-campaign-form.png
    :align: center
-   :alt: A dashboard showing the creation of a new marketing automation campaign in Odoo.
+   :alt: A blank marketing automation campaign form in Odoo Marketing Automation application.
 
-**Smart buttons**
+After entering a name for the marketing campaign, configure the target audience in the remaining
+fields.
 
-- :guilabel:`Templates`: represents the number of pre-configured mail templates being used in this
-  particular campaign. (Templates can always be created on-the-fly, as well).
-- :guilabel:`SMS`: represents the number of personalized SMS messages connected to this campaign.
-- :guilabel:`Clicks`: represents the number of times attached links have been clicked by recipients
-  of this campaign.
-- :guilabel:`Participants`: represents the number of contacts that have directly participated in
-  this campaign.
+A target audience can be configured by entering specific criteria for Odoo to use when determining
+to whom this marketing automation campaign should be sent. Creating specific target audiences can
+help businesses deliver tailored messages and campaigns that appeal to a niche demographic, thus
+providing more value and personalization to all subsequent communications.
 
-**Fields**
+In the :guilabel:`Target` field, use the drop-down menu to choose which model the target audience
+filters should be based on (e.g. :guilabel:`Contact`, :guilabel:`Lead/Opportunity`,
+:guilabel:`Sales Order`, etc.).
 
-- :guilabel:`Name`: represents the name of the marketing automation campaign being created.
-- :guilabel:`Target`: this field is a drop-down menu to choose which model is targeted by this
-  campaign (i.e., by Contacts, Sales Order, Lead/Opportunity, etc.).
-- :guilabel:`Filter`: this field provides numerous configurable criteria that can be used to further
-  narrow down the target recipients/audience for the marketing automation campaign.
+Select :guilabel:`Search More...` from the drop-down menu to reveal a :guilabel:`Search: Target`
+pop-up window containing all 157 target options.
 
-Campaign filters
-================
+Once a :guilabel:`Target` is selected, there's a :guilabel:`Unicity based on` field. This field is
+used to avoid duplicates based on the model chosen in the :guilabel:`Target` field.
 
-To add a :guilabel:`Filter` to the target audience, click :guilabel:`Add Filter`, and a node field
-appears. In the node field, a custom equation can be configured for Odoo to use when filtering who
-to include (and exclude) in this specific marketing campaign.
+.. example::
+   If :guilabel:`Customers` is chosen as the :guilabel:`Target`, select :guilabel:`Email` in the
+   :guilabel:`Unicity based on` field so Odoo doesn't process any records that have the same Email
```

Comment:
Is this what the sentence meant to convey? To me it wouldn't make sense for Odoo not to process *any* records with a shared email address
```suggestion
   :guilabel:`Unicity based on` field so Odoo only processes one record for each customer email
```